syncano/models/options.py

```diff
@@ -133,15 +133,23 @@ def get_endpoint_methods(self, name):
         endpoint = self.get_endpoint(name)
         return endpoint['methods']
 
-    def resolve_endpoint(self, name, properties):
-        endpoint = self.get_endpoint(name)
+    def resolve_endpoint(self, endpoint_name, properties, http_method=None):
+        if http_method and not self.is_http_method_available_for_endpoint(http_method, endpoint_name):
+            raise SyncanoValidationError(
+                'HTTP method {0} not allowed for endpoint name "{1}".'.format(http_method, endpoint_name)
```

Comment:
`HTTP method {0} not allowed for endpoint name` -> `HTTP method {0} not allowed for endpoint `